catalog-info.yaml

```
@@ -0,0 +1,55 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: go-libaudit
+  description: Go Licenser - Adds a license header to any Go source files
```

description from the about section of the repository:

```suggestion
  description: go-libaudit is a library for communicating with the Linux Audit Framework.
```